.github/workflows/ci.yml

```diff
@@ -1,4 +1,8 @@
-on: push
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: main
```

Comment:
I don't think there's any reason not to run CI on pull requests which don't target master. Especially since there's this playground branch that I believe we still have, which is also in production.